rhamt/scripting/config.py

```diff
@@ -0,0 +1,63 @@
+import click
+from yaml import safe_dump
+from yaml import safe_load
+
+from rhamt.utils.path import CONF_PATH
+
+ENV_LOCAL_CONF = CONF_PATH / "env.local.yaml"
+ENV_CONF = CONF_PATH / "env.yaml"
+
+
+@click.group()
+def main():
+    """RHAMT Configuration."""
+    pass
+
+
+@main.command(help="Create env.local.yaml file")
+@click.option(
+    "-h", "--hostname", default="http://localhost:8080", help="Hostname of the target application"
+)
+@click.option(
+    "-d",
+    "--webdriver",
+    default="Remote",
+    type=click.Choice(["Remote", "Chrome", "Firefox"]),
+    help="Webdriver",
+)
+@click.option(
+    "-w", "--wharf", default=None, help="Address of the wharf server for remote webdrivers"
+)
+@click.option(
+    "-e", "--executor", default="http://localhost:4444/wd/hub", help="Selenium Command executor"
+)
+@click.option(
+    "-b",
+    "--browser",
+    default="chrome",
+    type=click.Choice(["chrome", "firefox"]),
+    help="Browser name.",
+)
+@click.option("-fh", "--ftp-host", default=None, help="FTP server hostname")
+@click.option("-fu", "--ftp-username", default=None, help="FTP server username")
+@click.option("-fu", "--ftp-password", default=None, help="FTP server password")
```

Comment:
click option will be fp here ..

@click.option("-fp", "--ftp-password", default=None, help="FTP server password")